File: U1/PROYECTO/BanquitoServer_Soap_DotNet_GR01/BanquitoServer_Soap_DotNet_GR01/DTOs/SolicitudCreditoDTO.cs
using System.Runtime.Serialization;

namespace BanquitoServer_Soap_DotNet_GR01.DTOs
{
    /// <summary>
    /// DTO para solicitud de crédito
    /// </summary>
    [DataContract]
    public class SolicitudCreditoDTO
    {
        [DataMember]
        public string Cedula { get; set; }

        [DataMember]
        public decimal PrecioElectrodomestico { get; set; }

        [DataMember]
        public int NumeroCuotas { get; set; }
    }
}

File: U2/PROYECTO/BanquitoServer_Restfull_Java_GR01/src/main/java/ec/edu/monster/DTOs/SolicitudCreditoDTO.java
package ec.edu.monster.DTOs;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * DTO para solicitud de crédito
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SolicitudCreditoDTO {
    private String cedula;
    private String precioElectrodomestico;
    private String numeroCuotas;
}
